<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PostFeed</class>
 <widget class="QMainWindow" name="PostFeed">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>PostFeed</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLabel" name="token">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>51</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Token :</string>
    </property>
   </widget>
   <widget class="QLabel" name="facebookid">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>50</y>
      <width>21</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>ID :</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="tokenEdit">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>10</y>
      <width>331</width>
      <height>25</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="idEdit">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>40</y>
      <width>331</width>
      <height>25</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>299</x>
      <y>100</y>
      <width>91</width>
      <height>181</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Post</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="msgEdit">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>100</y>
      <width>231</width>
      <height>181</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="msgLabel">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>80</y>
      <width>57</width>
      <height>15</height>
     </rect>
    </property>
    <property name="text">
     <string>Message:</string>
    </property>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
